plugins/modules/public_cloud_flavorid.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: public_cloud_flavorid
+short_description: Get flavor id based on commercial name
+description:
+    - Get flavorId based on commercial name (t1-45, b2-7 etc)
+    - The flavorId change between region
+    - The retrieved flavorId can be used to spawn a new instance
+author: Synthesio SRE Team
+requirements:
+    - ovh >= 0.5.0
+options:
+    name:
+        required: true
+        description: The commercial name of the flavor (t1-45, b2-7 etc)
+    region:
+        required: true
+        description: The region where to lookup for flavorId
+    service_name:
+        required: true
+        description: The service_name
+
+'''
+
+EXAMPLES = '''
+- name: run installation
+  synthesio.ovh.public_cloud_flavorid
+    service_name: "{{ service_name }}"
```

Comment:
```suggestion
    service_name: "ns12345.ip-1-2-3.eu"
```
Use a real example without variable like we do in other modules

Reply:
I have both kind of examples :D